lib-ce/emqx_dashboard/test/emqx_dashboard_SUITE.erl

```diff
@@ -84,9 +88,93 @@ t_admins_add_delete(_) ->
     ?assertNotEqual(true, request_dashboard(get, api_path("brokers"),
         auth_header_("username", "pwd"))).
 
+t_admins_persist_default_password(_) ->
+    emqx_dashboard_admin:change_password(<<"admin">>, <<"new_password">>),
+    ct:sleep(100),
+    [#mqtt_admin{password=Password, tags= <<"administrator">>}] = emqx_dashboard_admin:lookup_user(<<"admin">>),
+
+    %% To ensure that state persists even if the process dies
+    application:stop(emqx_dashboard),
+    application:start(emqx_dashboard),
+
+    ct:sleep(100),
+
+    %% It gets restarted by the app automatically
+    [#mqtt_admin{password=PasswordAfterRestart}] = emqx_dashboard_admin:lookup_user(<<"admin">>),
+    ?assertEqual(Password, PasswordAfterRestart),
+    emqx_dashboard_admin:change_password(<<"admin">>, <<"public">>).
+
+debug(Label, Slave) ->
+    ct:print(
+      "[~p]~nusers local ~p~nusers remote: ~p~nenv local: ~p~nenv remote: ~p",
+      [
+       Label,
+       ets:tab2list(mqtt_admin),
+       rpc:call(Slave, ets, tab2list, [mqtt_admin]),
+       application:get_all_env(emqx_dashboard),
+       rpc:call(Slave, application, get_all_env, [emqx_dashboard])
+      ]).
+
+
+t_default_password_persists_after_leaving_cluster(_) ->
+    [#mqtt_admin{password=InitialPassword}] = emqx_dashboard_admin:lookup_user(<<"admin">>),
+
+    ct:print("Cluster status: ~p", [ekka_cluster:info()]),
```

Comment:
Nit: pal?

Reply:
I can see why ct:print is used, because there are quite a few ct:print in this module, will keep it.